### PR TITLE
[To rel/0.11] Change Query debug log for deletion and last

### PR DESCRIPTION
**Pull request by wshao08**
## Description
Add more deletion and Last query debug log to help for bug diagnostic.

### CODE REVIEW

**Review by JackieTien97 — APPROVED** [review]

File: server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java [hunk 1]
```diff
@@ -1633,6 +1632,7 @@ private void tryToDeleteLastCache(PartialPath deviceId, PartialPath originalPath
           if (lastPair != null && startTime <= lastPair.getTimestamp()
               && lastPair.getTimestamp() <= endTime) {
             ((MeasurementMNode) measurementNode).resetCache();
+            logger.info("[tryToDeleteLastCache] Last cache for path: " + measurementNode.getFullPath() + " is set to null");
```
Comment (wangchao316) [inline server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java:1635]: we could use placeholders for log-print...

Comment (HTHou) [inline server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java:1635]:
```suggestion
            logger.info("[tryToDeleteLastCache] Last cache for path: {} is set to null", measurementNode.getFullPath());
```

**Comment by jixuan1989** [comment]
what do you want in this PR?

File: server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java (hunk 1 above)
Comment (wshao08) [reply server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java]: Fixed

File: server/src/main/java/org/apache/iotdb/db/query/executor/LastQueryExecutor.java [hunk 2]
```diff
@@ -187,6 +187,7 @@ public QueryDataSet execute(QueryContext context, LastQueryPlan lastQueryPlan)
       TimeValuePair tvPair = cacheAccessors.get(i).read();
       if (tvPair != null) {
         resultContainer.add(new Pair<>(true, tvPair));
+        DEBUG_LOGGER.info("[LastQueryExecutor] Last cache hit for path: " + seriesPaths.get(i) + " with timestamp: " + tvPair.getTimestamp());
```
Comment (HTHou) [inline server/src/main/java/org/apache/iotdb/db/query/executor/LastQueryExecutor.java:190]:
```suggestion
        DEBUG_LOGGER.info("[LastQueryExecutor] Last cache hit for path: {} with timestamp: {}" , seriesPaths.get(i), tvPair.getTimestamp());
```

**Pull request merged**